### ISSUE TITLE
Document which `Units` variants are valid for each property

### ISSUE THREAD

**Issue by nicoburns** [vizia/morphorm#19]
I believe that not all variants of `Units` are valid for every style property which uses the `Units` type. It would be very helpful to have documentation stating which combinations and valid/invalid.

**Comment by geom3trik** [comment]
Agreed. I wonder if maybe new types should be used for those properties instead? The ones that come to mind are min/max constraints (where auto doesn't do anything), and border_ properties (where only pixels/percentage is used). In fact in vizia border is expressed with a `LengthOrPercentage` type which is then converted to `Units` during layout.
